components/layout/navigation.tsx

```diff
@@ -26,6 +28,11 @@ const navigation = [
     icon: QuestionMarkCircleIcon,
     href: "/faq",
   },
+  {
+    name: "WhatsApp",
+    icon: WhatsAppLogo,
+    href: "https://wa.me/6281257579812",
+  },
```

Comment:
```suggestion
  {
    name: "WhatsApp",
    icon: WhatsAppLogo,
    href: "https://wa.me/6281257579812",
    external: true,
  },
```

We need to add an `external` config so that we can switch between `next/link` and regular `<a>` tags if this key is set.